config/scribe.php

```diff
@@ -248,6 +248,9 @@
             Strategies\ResponseFields\GetFromResponseFieldAttribute::class,
             Strategies\ResponseFields\GetFromResponseFieldTag::class,
         ],
+        'examples' => [
+            Strategies\Examples\UseExampleTag::class,
```

Comment:
Nice. I feel like attributes are a more convenient way to specify multiple examples, so let's add support for that too.

Reply:
@shalvah let me know how you would like me to update the docs, should it be a new section under Eg: [Documenting your API](https://scribe.knuckles.wtf/laravel/documenting/#)

Reply:
Yeah, that makes sense.

Reply:
I'd say you can probably add it under the existing docs for requests and responses.